package.json

```diff
@@ -33,18 +33,18 @@
     "typescript",
     "lit"
   ],
-  "author": "The Polymer Authors",
+  "author": "Google LLC",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "lit": "^2.0.0-pre.1",
-    "lit-element": "^3.0.0-pre.2"
+    "lit": "^2.0.0-rc.1",
+    "lit-element": "^3.0.0-rc.1"
```

Comment:
We shouldn't need the extra dependency on lit-element, should we?

Reply:
+1, fixed